packages/services/src/services/Locale/Locale.js

```diff
@@ -85,6 +85,76 @@ const _axiosConfig = {
  */
 const _sessionListKey = 'dds-countrylist';
 
+/**
+ * Use the <html> lang attr to determine a return locale object
+ *
+ * @returns {object}
+ * @private
+ */
+const _getLocaleByLangAttr = () => {
+  if (root.document.documentElement.lang) {
+    const lang = root.document.documentElement.lang.toLowerCase();
+    if (lang.indexOf('-') === -1) {
+      return _localeDefault;
+    } else {
+      const codes = lang.split('-');
+      return { cc: codes[1], lc: codes[0] };
+    }
+  } else {
+    return _localeDefault;
+  }
+};
+
+/**
+ * Return a locale object based on the DDO API, or "false"
+ * so the consumer can decide what to do next
+ *
+ * @returns {(object | boolean)}
+ * @private
+ */
+async function _getLocaleFromDDO() {
+  const ddoLocal = await DDOAPI.getAll();
+
+  const lang = root.document.documentElement.lang || undefined;
+  let pageInfoIBM = ddoLocal.page.pageInfo.ibm;
+
+  if (ddoLocal && ddoLocal.page && ddoLocal.page.pageInfo) {
+    if (pageInfoIBM) {
+      // Set proper CC for us to use.
+      if (pageInfoIBM.country) {
+        pageInfoIBM.cc = pageInfoIBM.country.toLowerCase().trim();
+
+        // If there are multiple countries use just the first one for the CC value
+        if (pageInfoIBM.cc.indexOf(',') > -1)
+          pageInfoIBM.cc = pageInfoIBM.cc
+            .substring(0, pageInfoIBM.cc.indexOf(','))
+            .trim();
+
+        // Gb will be uk elsewhere
+        if (pageInfoIBM.cc === 'gb') pageInfoIBM.cc = 'uk';
+
+        // Map worldwide (ZZ) pages to US
+        if (pageInfoIBM.cc === 'zz') pageInfoIBM.cc = 'us';
+      }
+
+      if (lang) {
+        pageInfoIBM.lc = pageInfoIBM.lc || lang.substring(0, 2).toLowerCase();
+        pageInfoIBM.cc = pageInfoIBM.cc || lang.substring(3, 5).toLowerCase();
+      }
+
+      // Account for more incorrect coding.
+      if (!pageInfoIBM.lc) pageInfoIBM.lc = 'en';
+      if (!pageInfoIBM.cc) pageInfoIBM.cc = 'us';
```

Comment:
Let's kill this block (lines 140-147). It looks like if DDO fails, it will default to en/us. We should have en/us be the fallback if both `_getLocaleFromDDO` and `_getLocaleByLangAttr` fail.